CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,51 @@
 
 import Foundation
 
-print("Hello, World!")
+struct Person {
+    let name: String
+    var age: Int
+    var money: Int
+
+    mutating func buyCoffee(from memu: Coffee) {
```

Comment:
memu 에 오타가 존재하네요! 그리고 커피의 의미를 메뉴라는 단어로 지칭되는 게 애매해보여요.
메뉴는 포괄적인 의미가 강해서요!
```suggestion
    mutating func buyCoffee(_ coffee: Coffee) {
```

이렇게 전달인자 레이블로 언더바를 활용하시면, 해당 메서드를 호출할 때 중복되는 단어가 보이지 않기 때문에, 많이 활용합니다.
해당 부분은 좀 더 공부하시는 게 좋을 거 같아요~

Reply:
@jsa0224 변수이름이나 전달인자 레이블 공부좀 더해봐야겟네요!! 수정했습니다